score/scrabble_score.rb

```diff
@@ -0,0 +1,81 @@
+module Scrabble
+
+  class Scrabble
+    # Set constants with general error message, point values of each letter, all letters
+    WORD_ERROR = "ERROR: Invalid input. Either your word has more that 7 letters or you entered something ridiculous. You only have 7 tiles, and Scrabble uses WORDS!"
+    POINTS = {
+      1 => ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"],
+      2 => ["d", "g"],
+      3 => ["b", "c", "m", "p"],
+      4 => ["f", "h", "v", "w", "y"],
+      5 => ["k"],
+      6 => ["j", "x"],
+      7 => ["q", "z"]
+    }
+
+    LETTERS = ("a".."z").to_a
+
+    def self.get_points(word)
+      # set initial value of score
+      score = 0
+
+      # convert to string, make letters lower case, & push letters to array
+      word_array = word.downcase.chars
+
+      # for each letter in word_array, check POINTS and add to total to create score
+      word_array.each do |letter|
+        for num in 1..7
```

Comment:
It would be preferring if you used an `each` on the POINTS hash which will give you access to each key, value pair